src/lib/apollo/schema/client.graphql

```diff
@@ -18,11 +18,6 @@ extend type Silenced {
   deleted: Boolean!
 }
 
-extend type CheckConfig {
-  "Describes whether the check has been deleted from the system"
-  deleted: Boolean!
-}
```

Comment:
Was this intentional?

Reply:
Yeah, this is declared twice in the same file and triggers a graphql error when parsed. I'm not entirely sure why we're only encountering this now.

Reply:
This PR looks like the culprit https://github.com/sensu/web/commit/078d240ace2fc33dab20e103fecdfa4792084e30